docs/user/checks.rst

```diff
@@ -269,6 +269,314 @@ ECMAScript template literals
    :ref:`check-formats`,
    `Template literals <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals>`_
 
+.. _check-fluent-source-syntax:
```

Comment:
The skeleton of the documentation should be generated by `weblate list_checks` to ensure consistency.